sw/simulation/environment.cpp

```diff
@@ -14,7 +14,17 @@ using namespace std;
 
 Environment::Environment(void)
 {
+  generate_dungeon();
+
+  string s = param->agent_initialization();
+  if (!strcmp(s.c_str(), "in_area")){
+    complete_folder();
+  }
+  load_gas_data();
```

Comment:
What if the user doesn't want gas data? Could we make this the same as food below, dependent on the fitness function?